### PR TITLE
CMOS-69: component doc for alertmanager, grafana, jaeger, prometheus

### PR DESCRIPTION
**Pull request by shovanmaity**

### CODE REVIEW

**Review by maria-robobug — CHANGES_REQUESTED** [review]
Looks good, don't know enough about CMOS to comment on features, but added comments on what I could review.

File: docs/modules/ROOT/pages/component-alertmanager.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+As part of the Couchbase Monitoring and Observability Stack (CMOS), Alertmanager comes with a set of built-in standard rules to handle alerts generated for Couchbase Server and other CMOS components.
+
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+image:component-alertmanager.png[]
+
+Based on a predefined set of rules, the CMOS Prometheus Server generates alerts depending on the metrics it collects from Couchbase Server and other targets.
+By default, these alerts are  forwarded to CMOS Alertmanager.
+The Alertmanager can be configured to route these alerts to a user defined receiver integration such as e-mail, slack, pagerduty or other API-based endpoints to notify the user.
```
[the remark this answers is not in the document]
Comment (RichardSmedley) [reply docs/modules/ROOT/pages/component-alertmanager.adoc:21]: Although not on the word-list - https://docs.couchbase.com/home/contribute/word-list.html
you will see that it is similar to other examples such as data-driven

**Review by maria-robobug — APPROVED** [review]
Just added some additional minor comments, but 👍 LGTM

File: docs/modules/ROOT/pages/component-alertmanager.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+As part of the Couchbase Monitoring and Observability Stack (CMOS), Alertmanager comes with a set of built-in standard rules to handle alerts generated for Couchbase Server and other CMOS components.
+
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+image:component-alertmanager.png[]
+
+Based on a predefined set of rules, the CMOS Prometheus Server generates alerts depending on the metrics it collects from Couchbase Server and other targets.
+By default, these alerts are forwarded to CMOS Alertmanager.
+The Alertmanager can be configured to route these alerts to a user-defined receiver integration such as e-mail, slack, PagerDuty or other API-based endpoints to notify the user.
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/component-alertmanager.adoc:25]: Slack is a brand name and should (I think) be capitalised
```suggestion
The Alertmanager can be configured to route these alerts to a user-defined receiver integration such as e-mail, Slack, PagerDuty or other API-based endpoints to notify the user.
```

File: docs/modules/ROOT/pages/component-alertmanager.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+== Get Started
+
+In CMOS, the Alertmanager is included and enabled by default.
+It ships with a set of standard built-in rules to de-duplicate and group alerts based on multiple alert parameters.
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/component-alertmanager.adoc:30]: I don't think we currently ship de-duplication rules in our stock config.

File: docs/modules/ROOT/pages/component-alertmanager.adoc [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+Additionally, it can be customized to send notifications to different https://prometheus.io/docs/alerting/configuration[receiver^] endpoints.
+
+ 
+The CMOS Alertmanager can be deployed xref:tutorial-onpremise.adoc[on premise] or using xref:tutorial-kubernetes.adoc[kubernetes].
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/component-alertmanager.adoc:34]: Is Kubernetes a proper noun? (cc @maria-robobug)

```suggestion
The CMOS Alertmanager can be deployed xref:tutorial-onpremise.adoc[on premise] or using xref:tutorial-kubernetes.adoc[Kubernetes].
```

File: docs/modules/ROOT/pages/component-alertmanager.adoc [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Check out the reference xref:architecture.adoc[Couchbase Monitoring Observability Stack](CMOS) architecture to learn more.
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/component-alertmanager.adoc:35]:
```suggestion
Check out the reference xref:architecture.adoc[Couchbase Monitoring and Observability Stack] (CMOS) architecture to learn more.
```

File: docs/modules/ROOT/pages/component-grafana.adoc [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+A dynamic cloud environment makes it essential to visualize the state of the system at all times.
+As part of the Couchbase Monitoring and Observability Stack (CMOS), Grafana has a set of dashboards that allow administrators to query the state of the system centrally and take action at any given time.
+
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+image:component-grafana.png[]
+
+In CMOS, Grafana is linked to data sources such as https://prometheus.io[Prometheus^] and https://grafana.com/Loki[Loki^].
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/component-grafana.adoc:19]: NITPICK: would it make sense for these to be links to the relevant component docs, now that we have those pages?

**Review by markspolakovs — APPROVED** [review]
Technical content LGTM, just a few comments.

File: docs/modules/ROOT/pages/component-alertmanager.adoc (hunk 4 above)
Comment (maria-robobug) [reply docs/modules/ROOT/pages/component-alertmanager.adoc:34]: Believe so, should be capitalized as you suggested 👍 .

File: docs/modules/ROOT/pages/component-grafana.adoc (hunk 6 above)
Comment (shovanmaity) [reply docs/modules/ROOT/pages/component-grafana.adoc:19]: We don't have component configuration page yet, once we have those, will update the links.

**Pull request merged**